flixel/input/actions/FlxActionInputDigital.hx

```diff
@@ -307,6 +308,34 @@ class FlxActionInputDigitalSteam extends FlxActionInputDigital
 	}
 }
 
+class FlxActionInputDigitalAndroid extends FlxActionInputDigital
+{
+	/**
+	 * Android buttons action input for the BACK and MENU buttons on Android
+	 * @param	androidKeyID Key identifier (FlxAndroidKey.BACK, FlxAndroidKey.MENU... those are the only 2 android specific ones)
+	 * @param	Trigger What state triggers this action (PRESSED, JUST_PRESSED, RELEASED, JUST_RELEASED)
+	 */
+	public function new(androidKeyID:FlxAndroidKey, Trigger:FlxInputState)
+	{
+		super(MOBILE, androidKeyID, Trigger);
+	}
+
+	override public function check(Action:FlxAction):Bool
+	{
+		return switch (trigger)
+		{
+			#if android
+			case PRESSED: FlxG.android.checkStatus(inputID, PRESSED) || FlxG.android.checkStatus(inputID, PRESSED);
```

Comment:
This line is checking the same thing twice, `pressed || pressed`. Looking at other classes in this module, it seems like this should've been `pressed || justPressed`?